Vagrantfile

```diff
@@ -101,18 +139,10 @@ Vagrant.configure("2") do |config|
 
   # Mount some directories from host in guest if they exist.
   ## Place to store your "Dev" stuff.
-  require 'yaml'
-
-  config_file = File.dirname(File.expand_path(__FILE__)) + "/config.yaml"
-
-  if File.exist?(config_file)
-    kdk_dev_folder = YAML.load_file(config_file)['kdk_dev_folder']
-  else
-    kdk_dev_folder = "Dev"
-  end
 
-  if File.directory?(File.expand_path("~/" + kdk_dev_folder))
-    config.vm.synced_folder "~/" + kdk_dev_folder, "/home/vagrant/Dev"
+  host_dev_folder = if custom != nil && custom.key?('host_dev_folder') then custom['host_dev_folder'] else "~/Dev" end
+  if File.directory?(File.expand_path(host_dev_folder))
+    config.vm.synced_folder host_dev_folder, "/home/vagrant/Dev"
```

Comment:
NIT: is it possible to use the same name as the custom `host_dev_folder` rather than default to `Dev`?

Reply:
@rtluckie I'm going to merge this PR, and then follow-on with a PR that makes the host_dev_folder a list of from_host_dirs to_guest_dirs mounts.  I'll also change the config.yaml.example to be config.yaml.defaults, which can still be overridden via a config.yaml.  Then I can remove some of the logic to check that config.yaml exists ,and remove the default hard coding in this file.  Cool?

Reply:
sgtm

Reply:
Thanks @rtluckie .  Merging then.  Will follow-on a PR

Reply:
@rtluckie  See this follow-on PR

https://github.com/cisco-sso/k8s-devkit/pull/13